pages/development/pushing-code.mdx

```diff
@@ -1,8 +1,48 @@
 # Pushing code to your workspace
 
-After defining your data models and components, you need to push them to your Embeddable workspace to make them available in the dashboard builder.
+To make your [data models](/data-modeling/defining-models) and [components](/development/defining-components) available in the dashboard builder, you need to push them to your Embeddable Workspace.
 
-### Push command
+You can do this by running the [**push command**](/development/pushing-code#push-command), which, by default, pushes both data models and components. 
+
+## Controlling what gets pushed
+
+If you store your models and components in [separate repos](/data-modeling/getting-setup#2-storing-models-separately), you can push them independently. 
+
+In your component repo, in `embeddable.config`, simply set the following:
```

Comment:
@hjkmarshall this is great, but it looks like it's copied and pasted from `testing-models.mdx`.  I think this will quickly make these docs hard to maintain if we repeat the same info in multiple places.

Suggestion:
1. either put it in one page, and link to it in other places, or
2. create a component holding this information, and then import that component in both places

Reply:
Ignore this.  I'm mistaken